fix: Use types instead of interfaces (#274)
SAP/ai-sdk-js: .changeset/interface-vs-type.md

---
'@sap-ai-sdk/orchestration': minor
---

> [Compatibility Note] Switch some of the orchestration interfaces to types, which was introduced by accident. 

Reviewer: ```suggestion
[Compatibility Note] Switch some of the orchestration interfaces to types, as they were introduced by accident.
```
